packages/calcite-components/src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -1851,6 +1852,64 @@ describe("calcite-input-date-picker", () => {
     expect(await calendar.isVisible()).toBe(false);
     expect(await isElementFocused(page, "#input")).toBe(true);
   });
+
+  describe("theme", () => {
+    describe("default", () => {
+      themed(html`<calcite-input-date-picker range></calcite-input-date-picker>`, {
+        "--calcite-input-date-picker-background-color": {
+          targetProp: "--calcite-input-date-picker-background-color",
+        },
+        "--calcite-input-date-picker-border-color": {
+          targetProp: "--calcite-input-date-picker-border-color",
+        },
+        "--calcite-input-date-picker-shadow": {
+          targetProp: "--calcite-input-date-picker-shadow",
+        },
+        "--calcite-input-date-picker-divider-background-color": {
+          targetProp: "--calcite-input-date-picker-divider-background-color",
+        },
+        "--calcite-input-date-picker-icon-color": {
+          targetProp: "--calcite-input-date-picker-icon-color",
+        },
+        "--calcite-input-date-picker-icon-color-hover": {
+          targetProp: "--calcite-input-date-picker-icon-color-hover",
+        },
+      });
+    });
+
+    describe("input", () => {
+      themed(html`<calcite-input-date-picker range></calcite-input-date-picker>`, {
```

Comment:
Can these tokens be merged into a single `themed` test? The input HTML is the same for both. 

I do think we need a separate test group for the non-range configuration.